app/views/HomePage/TopSellingItem/index.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from 'react';
+import { Link } from '@the-deep/deep-ui';
+import {
+    gql,
+    useQuery,
+} from '@apollo/client';
+import Carousel from '#components/Carousel';
+import CarouselItem from '#components/Carousel/CarouselItem';
+import CarouselButton from '#components/Carousel/CarouselButton';
+import { TopSellingItemQuery } from '#generated/types';
+import routes from '#base/configs/routes';
+
+import styles from './styles.css';
+
+const TOP_SELLING_ITEMS = gql`
+    query TopSellingItem {
+        books(ordering: "ordered_count", pageSize: 5) {
+            results {
+                image {
+                    name
+                    url
+                }
+                title
+                price
+                categories {
+                    name
+                }
+                numberOfPages
+            }
+        }
```

Comment:
include `id` field when querying for books. `id` field is used to cache the results by graphQL.

Reply:
Done